word_guess.rb

```diff
@@ -0,0 +1,213 @@
+
+  frog3 = "       @ . . @
+      (-------)
+    (   >___<   ) ".colorize(:green) +
+    "\n   ~~~~~~~~~~~~~~
+    ~~~~~~~~~~~~~~
+    ~~~~~~~~~~~~~~
+    ~~~~~~~~~~~~~~".colorize(:blue)
+
+  frog4 = "       @ . . @
+      (-------)".colorize(:green) +
+      "\n   ~~~~~~~~~~~~~~
+      ~~~~~~~~~~~~~~
+      ~~~~~~~~~~~~~~
+      ~~~~~~~~~~~~~~
+      ~~~~~~~~~~~~~~ ".colorize(:blue)
+
+  frog5 = "       @ . . @".colorize(:green) +
+  "\n   ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~ ".colorize(:blue)
+
+  frog6 =  "  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~ ".colorize(:blue)
+
+  frogarray = []
+  frogarray.push(frog1, frog2, frog3, frog4, frog5, frog6)
+#Create an array of possible words for the player to guess. Use sample to get one word from array.
+mystery_words = %w(get red kitty nose)
+
+  puts "Welcome to Word Guess.\nYour goal is to guess the correct word.\nEach time you guess a wrong letter not in the mystery word, the frog will sink into the water a little more."
+  puts "You must guess the correct word before the frog is submerged."
+  puts "Let's get started!"
+  #Print ASCII art and mystery word.
+  puts frogarray[0]
+  # guesses = false
+  # until guesses == words_display ||
+  # sample is a method on the array class
+  # chars is a method on the string class and returns an array of characters
+  words_display = mystery_words.sample.chars
+  # print words_display
+  # replace the letter in mystery word with dashes
+  n = words_display.length
+  dashed_word = ("-" * n).chars.to_a
+  p dashed_word
+
+
+  # index where they have guessed
+  #guessed_letter = []
+  wrong_guesses = []
+  # display for each turn
+  # store the wrong guess in an array
+  while dashed_word.include?("-") && wrong_guesses.length < 5
+    puts "What letter do you want to guess?"
+    guessed_letter = gets.chomp
+    if words_display.include?(guessed_letter)
+      puts "That is correct!"
+      puts frogarray[wrong_guesses.length]
+      # puts frogarray[g]
+      # where is the guessed_letter in words_display
+      # replace the dash (a string) at that index with guessed_letter
+      # ["r", "e", "d"].index("r")
+      # => 0
+      # a = words_display.index(guessed_letter)
+      # puts a
+      # a = 0
+      # dashed_word[a] = guessed_letter
+      a = []
+      words_display.each_with_index do |val, ind| # this method will keep looping through every thing in the array until gets all
+        if val == guessed_letter # "r"
+           a.push(ind) # 0
+        end
+          a.each do |i|
+            dashed_word[i] = guessed_letter
+        end
+      end
+
+    elsif wrong_guesses.include?(guessed_letter) && dashed_word.include?(guessed_letter)
+      puts "You have already guessed this letter, please guess again."
+          puts frogarray[wrong_guesses.length]
+    else puts "No! #{guessed_letter} is wrong\n"
```

Comment:
Convention says that we should put the contents of the `else` statement on the line after the `else`, so this would be:
````
else
   puts "No! #{guessed_letter} is wrong\n"
```
etc